app/main.ts

```diff
@@ -3008,7 +3008,7 @@ ipc.handle('show-save-dialog', async (_event, { defaultPath }) => {
   const { canceled, filePath: selectedFilePath } = await dialog.showSaveDialog(
     mainWindow,
     {
-      defaultPath,
+      defaultPath: join(app.getPath('downloads'), defaultPath),
```

Comment:
Here I changed the original behavior of the save dialog for a single attachment, because I think it makes sense to use the users download directory as a base directory.
Otherwise, when I try to save a file, it uses `./` as the default directory and I have to navigate to downloads manually.

Reply:
On at least some platforms, providing nothing here allows the user to choose whatever location they want, and it will save that location between different attachment saves. We may want to revert this change.

Reply:
Okay, makes sense.
So should I revert it?

Reply:
Yes, please.